api/src/certification/enrolment/infrastructure/repositories/candidate-repository.js

```diff
@@ -70,12 +71,28 @@ export async function update(candidate) {
   }
 
   await knexConn('certification-subscriptions').where({ certificationCandidateId: candidate.id }).del();
+
   for (const subscription of candidate.subscriptions) {
-    await knexConn('certification-subscriptions').insert({
-      certificationCandidateId: candidate.id,
-      type: subscription.type,
-      complementaryCertificationId: subscription.complementaryCertificationId,
-    });
+    if (subscription.isCore()) {
+      await knexConn('certification-subscriptions').insert({
+        certificationCandidateId: candidate.id,
+        type: SUBSCRIPTION_TYPES.CORE,
```

Comment:
des fois il y a usage du type (par exemple ligne 127 plus bas), des fois il y a usage de la constant (comme ici). J'opterai pour avoir une coherence (et eventuellement sortir une fonction pour ce code qui se repete)
 J'utiliserai le type pour ma part car je prefere sauvegarder le domain model :) 

```suggestion
        type: subscription.type,
``` ca